administration/tests/test_admin_profiles.py

```diff
@@ -22,27 +22,28 @@ def test_get_profiles_not_authorized(self):
 
     def test_get_profile_id_not_authorized(self):
         response = self.client.get(
-            path="/api/admin/profiles/" + str(self.user.id) + "/"
+            path="/api/admin/profiles/" + str(self.company.id) + "/"
         )
         self.assertEqual(status.HTTP_401_UNAUTHORIZED, response.status_code)
 
-    def test_get_profiles(self):
+    def test_get_profiles_structure_json(self):
```

Comment:
We always have json in response, you don't need to emphasise it in test name, it's better to use '_authenticated_is_not_admin', '_authenticated_is_admin', etc.

Reply:
Done